src/Pages/Room/roomDetail.js

```diff
@@ -0,0 +1,80 @@
+import { useParams, useNavigate } from 'react-router-dom';
+import { useEffect, useState } from 'react';
+import axios from 'axios';
+import './RoomDetail.css';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import { faCalendarCheck } from '@fortawesome/free-regular-svg-icons';
+import { faCircleChevronRight } from '@fortawesome/free-solid-svg-icons';
+
+const RoomDetail = () => {
+  const [roomDetail, setRoomDetail] = useState([]);
+  // const [message, setMessage] = useState({});
+  const { id } = useParams();
```

Comment:
- [ _Opional_ ]: Please make sure to remove all unnecessary code from the project.